### PR TITLE
A first draft of PR templates for rosdistro

### PR DESCRIPTION
**Pull request by tfoote**
I hope that these templates can help accelerate our review process. These are minimum sketches. There's lots of space to add more information, but I wanted to start the process and we can extend it further.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/rosdep_pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  Ubuntu: https://packages.ubuntu.com/
+    - REQUIRED
+  Fedora: https://apps.fedoraproject.org/
+    - If AVAILABLE
+  Arch: https://www.archlinux.org/packages/
+    - IF AVAILABLE
+  Gentoo: https://packages.gentoo.org/packages/
+    - IF AVAILABLE
+  macOS: https://formulae.brew.sh/
+    - IF AVAILABLE
```
Comment (cottsay) [inline .github/PULL_REQUEST_TEMPLATE/rosdep_pull_request_template.md:32]: Formatting is lost on this section when the markdown is rendered.

File: .github/PULL_REQUEST_TEMPLATE/rosdep_pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Distro packaging links:
+
+## Links to Distribution Packages
+
+  Debian: https://packages.debian.org/
+    - REQUIRED
+  Ubuntu: https://packages.ubuntu.com/
+    - REQUIRED
+  Fedora: https://apps.fedoraproject.org/
```
Comment (cottsay) [inline .github/PULL_REQUEST_TEMPLATE/rosdep_pull_request_template.md:25]:
```suggestion
  Fedora: https://apps.fedoraproject.org/packages/
```

File: .github/PULL_REQUEST_TEMPLATE/rosdep_pull_request_template.md (hunk 1 above)
Comment (tfoote) [reply .github/PULL_REQUEST_TEMPLATE/rosdep_pull_request_template.md]: Fixed.

File: .github/PULL_REQUEST_TEMPLATE/rosdep_pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+Distro packaging links:
+
+## Links to Distribution Packages
+
+<!-- Replace the REQUIRED areas and state not available for IF AVAILABLE -->
+
+- Debian: https://packages.debian.org/
+  - REQUIRED
+-  Ubuntu: https://packages.ubuntu.com/
```
Comment (cottsay) [inline .github/PULL_REQUEST_TEMPLATE/rosdep_pull_request_template.md:25]:
```suggestion
- Ubuntu: https://packages.ubuntu.com/
```

**Review by cottsay — APPROVED** [review]

**Comment by 130s** [comment]
Is this template supposed to appear when one clicks "Create a pull request"? If so, the text area in that view is still blank for me. Tested in https://github.com/ros/rosdistro/compare/master...bloom-py_trees_js-2

Seeing a template is very helpful as I won't have to click and open another page to find the PR guidance.

**Pull request merged**